### DISCUSSION TITLE
Cannot exec any brew command(s) anymore after update

### DISCUSSION THREAD

**Discussion by MxHex** [Homebrew/discussions#1104] [Everyday usage] [answered]
Brew upgraded today to 
**HOMEBREW_VERSION: 3.0.8-6-ga235b38
ORIGIN: https://github.com/Homebrew/brew
HEAD: a235b38000d06b6732dcf1d5deab7ad57710ad11**

Now nearly all of the command throw errors, e.g.:

>▶ brew upgrade
>Error: CLI arg for `HEAD?` is not declared for this command

or

> brew install <ANY_PACKAGE>
> Error: CLI arg for `universal?` is not declared for this command

Both tell me to "report this issue", so here we go:

> Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/cli/args.rb:147:in `method_missing'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:127:in `prevent_build_flags'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:179:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:122:in '< main > '

**Comment by Bo98** [comment, marked as answer]
Fixed in https://github.com/Homebrew/brew/pull/10904.
